ui/raidboss/data/04-sb/trial/rathalos-ex.ts

```diff
@@ -69,6 +70,7 @@ const triggerSet: TriggerSet<Data> = {
           // TODO: we could be fancier and say "Run through" or something for #2.
           en: 'Left Flank (or out)',
           de: 'Linke Flanke (oder raus gehen)',
+          fr: 'Suivez le flanc gauche (ou extérieur)',
```

Comment:
Here (normal and ex) I took into account your note and also the mechanics. The boss flicks his tail twice while turning twice, I find it more consistent to say to follow his flank.

Reply:
Thanks for the note.  I think adding a "follow" verb here is fine, although the player will be instructed to follow twice.  In English, I thought it made the most sense to start with the most important word ("left").